content/docs/command-reference/exp/run.md

```diff
@@ -90,8 +90,8 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
   [example](#example-run-a-grid-search)).
 
 - `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
-  default one will be generated otherwise, such as `exp-f80g4` (based on the
-  experiment's hash).
+  default one will be generated otherwise, such as `puffy-daks`. The name of the
+  experiment is exposed as an environment variable called `DVC_EXP_NAME`.
```

Comment:
This is just a very brief mention of the env var. Given that https://github.com/iterative/dvc/issues/6668 generated some interest, perhaps is worth to add an example using the env var somewhere to increase visibility

Reply:
Note in `exp save -n` too, please (see admon suggestion too).

Reply:
> example using the env var somewhere

IMO the ref is already kinda long, so no need. This seems like an edge case for people who would be able to figure it out anyway (as long as they know there's an env var).